x/auth/ante/ante.go

```diff
@@ -22,6 +22,7 @@ func NewAnteHandler(
 		TxTimeoutHeightDecorator{},
 		NewValidateMemoDecorator(ak),
 		NewConsumeGasForTxSizeDecorator(ak),
```

Comment:
I don't think we need this ante handler (which is named as if it works, but doesn't.... that is misleading.

I would just add the "reject if granted is set" inside the normal fee decorator for now, and can add a new decorator when there is real logic.

Reply:
How about calling it `RejectFeeGranterDecorator`? That could still be useful for chains that don't enable fee grants and I'd prefer to leave a placeholder than muck around with other ante decorators.

Reply:
Sounds fine with a clear name.
My point is that the deduct fee middle ware will need to change to support fee grants, so why not just have 2 such middlewares. A no grant version (now) and a grant supporting one (can swap out later)